core/templates/services/speech-synthesis-chunker.service.ts

```diff
@@ -187,9 +190,11 @@ export class SpeechSynthesisChunkerService {
     elt.find('oppia-noninteractive-' + this.RTE_COMPONENT_NAMES.Math)
       .replaceWith(function() {
         var element = <HTMLElement><any> this;
-        if (element.attributes['raw_latex-with-value'] !== undefined) {
+        if (element.attributes['math_content-with-value'] !== undefined) {
+          var mathContent = _this.htmlEscaper.escapedJsonToObj(
+            element.attributes['math_content-with-value'].textContent);
           const latexSpeakableText = _this._formatLatexToSpeakableText(
-            element.attributes['raw_latex-with-value'].textContent);
+            (mathContent as any).raw_latex);
```

Comment:
Don't use any I think we have a class for these object can you please check?

Reply:
done , I've added a custom type.